tools/depends/native/python3/Makefile

```diff
@@ -7,12 +7,16 @@ LIBNAME=Python
 VERSION=3.7.3
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.xz
+CWD=$(shell pwd)
+HOSTPYTHONDIR=$(CWD)/$(PLATFORM)/hostpython
+BASE_URL=https://www.python.org/ftp/python/$(VERSION)
 
-CONFIGURE=./configure --prefix=$(NATIVEPREFIX) --disable-shared --disable-framework --without-ensurepip --without-pymalloc --with-system-ffi
+CONFIGURE=./configure --prefix=$(NATIVEPREFIX) --disable-shared --disable-framework --without-pymalloc
```

Comment:
This conflicted, can you review @Rechi

Reply:
Why remove ` --with-system-ffi`, which was added in master branch?

Reply:
Should be resolved with the latest version